pkg/component/monitoring/prometheus/seed/podmonitors.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package seed
+
+import (
+	_ "embed"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CentralPodMonitors returns the central PodMonitor resources for the seed prometheus.
+func CentralPodMonitors() []*monitoringv1.PodMonitor {
+	return []*monitoringv1.PodMonitor{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "garden",
+			},
+			Spec: monitoringv1.PodMonitorSpec{
+				// Selector: metav1.LabelSelector{MatchExpressions: []metav1.LabelSelectorRequirement{
+				// 	{Key: "prometheus.io/scrape", Values: []string{"true"}, Operator: metav1.LabelSelectorOpIn},
+				// 	{Key: "prometheus.io/port", Operator: metav1.LabelSelectorOpExists},
+				// }},
```

Comment:
We cannot use it yet since we wrongly use annotations instead of labels. If we used labels, this was the correct way of configuring it, hence I kept it (commented out) and added the TODO statement accordingly.